imodels/rule_set/rule_set.py

```diff
@@ -0,0 +1,72 @@
+from typing import Tuple, List, Dict
+
+import numpy as np
+import pandas as pd
+from sklearn.utils.validation import check_array, check_is_fitted
+
+
+class RuleSet:
+
+    @staticmethod
+    def _enum_features(X, feature_names: List[str]) -> Tuple[List[str], Dict[str, str]]:
+        enum_feature_names = [f'feature_{i}' for i in range(X.shape[1])]
+        if feature_names is None:
+            feature_names = enum_feature_names
+        feature_dict = {k: v for k, v in zip(enum_feature_names, feature_names)}
+        return feature_names, feature_dict
+
+    def _get_tree_ensemble(self):
```

Comment:
This is great for now - going forward, we might want this to be more generic, so something like `_mine_rules_with_model` - this would be for similar rulesets where our initial rules comes from something like apriori rather than a tree ensemble.

Reply:
sounds good!